base/kra/src/main/java/org/dogtagpki/server/kra/rest/KeyService.java

```diff
@@ -629,15 +630,20 @@ public KeyInfo createKeyDataInfo(KeyRecord rec, boolean getPublicKey) throws EBa
         return ret;
     }
 
-    private String createSearchFilter(String status, String clientKeyID, String realm) {
+    private String createSearchFilter(String status, String clientKeyID, String realm, String ownerName) {
         String filter = "";
         int matches = 0;
 
-        if ((status == null) && (clientKeyID == null)) {
+        if ((status == null) && (clientKeyID == null) && (ownerName == null)) {
             filter = "(serialno=*)";
             matches ++;
         }
 
+        if (ownerName != null) {
+            filter = "(keyownerName=" + LDAPUtil.escapeFilter(ownerName) + ")";
```

Comment:
Let's use the same capitalization, i.e. `keyOwnerName`, as defined here:
https://github.com/dogtagpki/pki/blob/master/base/server/src/main/java/com/netscape/cmscore/dbs/KeyRecord.java#L54

Reply:
yeah. ok fixed.